Samples/resource-owner-password-credential/Angular2/Angualar2-Client-ROPC/app/authorize/authoriza-service.ts

```diff
@@ -0,0 +1,72 @@
+﻿import {Injectable} from '@angular/core';
+import {Http, Response, Headers, RequestOptions} from '@angular/http';
+import {logModel, registerModel, token,Regresult} from './authorize-component'
+import {Configuration} from '../app.constants'
+import {Observable} from 'rxjs/Rx';
+import 'rxjs/add/operator/map';
+@Injectable()
+export class authervice {
+    constructor(private http: Http, private app: Configuration) { }
+  
+    private _authUrl = this.app.Server;  // URL to web api
+    private headers = new Headers({ 'Content-Type': 'application/X-www-form-urlencoded' });
+    private jheaders = new Headers({ 'Content-Type': 'application/json' });
+    private authheaders;
+    private options = new RequestOptions({ headers: this.headers });
+    private joptions = new RequestOptions({ headers: this.jheaders });
+    private Authoptions;// = new RequestOptions({ headers: this.authheaders });
+    private tokenParams = "grant_type=password" +// password type reuqets with credentials read more on grant_types
+                          "&client_id=localApp" +
+                          "&resource="+ this.app.FileServer + // audience url . read more on docs /blog
+                          "&responseType=token" + // get token 
+                          "&scope=offline_access profile email roles"; // offline_access for refresh_token read more on docs / blog
+
+    getUserInfo() {
```

Comment:
nit: this logically isn't a AuthService responsibility so consider moving to a separate service (i.e. ResourceService)